Formula/neomutt.rb

```diff
@@ -23,6 +22,7 @@ class Neomutt < Formula
   def install
     ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
     system "./configure", "--prefix=#{prefix}",
+                          "--disable-doc",
```

Comment:
This disables building manpages.

Reply:
I know the building the whole neomutt was failing because of building docs was failing. But the issue wasn't in neomutt, but in docbook or docbook-xsl Formulas. Fixed here:
8c92df0b3bfc24c2506e72ee57856f54ca741e46
1aba4581ef39a7c3df3dd7f91e70400f61d5ddd4